docs/search relevance/README.md

```diff
@@ -24,34 +24,43 @@ There's a lot of interesting discussion to be had about different search modalit
 
 ### Metrics
 
 We're using multiple complementary metrics to measure search relevance:
 
 #### Explicit
 
-- **Normalised Discounted Cumulative Gain (NDCG)** is well explained in [this post](https://www.ebayinc.com/stories/blogs/tech/measuring-search-relevance/) by ebay. It relies on explicitly collected data from users who know that the data they give us will be used to measure algorithmic performance. It compares the actual performance of a query-type (the order of a set of rated results) to the _ideal_ performance (the same results, sorted from most to least relevant).
+- **Normalised Discounted Cumulative Gain (NDCG)** is well explained in [this post](https://www.ebayinc.com/stories/blogs/tech/measuring-search-relevance/) by ebay. It relies on explicitly collected data from users who know that the data they give us will be used to measure algorithmic performance. It compares the actual performance of a query-type (the order of a set of rated results) to the _ideal_ performance (the same results, sorted from most to least relevant according to the user's rating).
 - **Strict, loose, and permissive precision** use the same data as NDCG. For strict precision, the percentage of results rated 4+ is counted. Loose precision counts the percentage of 3+ ratings, and permissive precision counts 2+. Explained in the book [_Search Analytics For Your Site_](https://rosenfeldmedia.com/books/search-analytics-for-your-site/) by Louis Rosenfeld.
 
 #### Implicit
 
+Among many others, we're looking at:
+
 - **Click through rate (CTR)** is measured passively by tracking users' behaviour while they use the search function. This version of CTR is different to the usual definition; we take the ratio of the number of distinct searches to the number of items clicked on for each anonymised session id.
-- **Top 5 click through rate (CTR5)** is almost exactly the same as the above, but only counts the clicks on works which appear in the top 5 results.
+- **Top n click through rate (CTR5)** is almost exactly the same as the above, but only counts the clicks on works which appear in the top n results.
+- **Click distribution curve fitting** uses plots of the distribution of clicks on the first page of results. This distribution tends to follow a regular exponentially decaying shape, but search variants produce slightly different behaviour. In the simplest case of motivation for a test (wanting to generate more clicks at the top of the set of results), observing differences in the graphs over a sufficient window of time will reveal which variant is performing better. This can also be quantified by fitting a line of the form `y = (a * e^(-b * x)) + c` to the curves and comparing the values of `b`. A lower `b` corresponds to a sharper elbow in the decay, and therefore a steeper concentration of clicks towards the top of the result set.
```

Comment:
Could we have a labelled graph showing what we mean by this? This is purely selfish as I work better with being pointed to the visual thing we're talking about.